src/tile/tile.spec.ts

```diff
@@ -1,9 +1,144 @@
+  findSiblings,
+  findTile,
+  renderTile,
+} from "./tile";
+import { Board } from "../board/board";
+import { BLACK_PLAYER } from "../player/player";
+import { line } from "blessed";
+
+const board: Board = [
+  [
+    undefined,
+    undefined,
+    undefined,
+    { symbol: "B", color: "cyan" },
+    { symbol: "A", color: "green", playedBy: BLACK_PLAYER, lastPlayed: false },
+    { symbol: "D", color: "cyan" },
+    { symbol: "A", color: "red" },
+    undefined,
+    undefined,
+    undefined,
+  ],
+  [
+    undefined,
+    undefined,
+    { symbol: "E", color: "magenta" },
+    { symbol: "F", color: "blue" },
+    { symbol: "C", color: "green", playedBy: BLACK_PLAYER, lastPlayed: false },
+    { symbol: "E", color: "green" },
+    { symbol: "D", color: "green" },
+    undefined,
+    undefined,
+  ],
+  [
+    undefined,
+    { symbol: "C", color: "cyan" },
+    { symbol: "D", color: "blue" },
+    { symbol: "C", color: "yellow", playedBy: BLACK_PLAYER, lastPlayed: false },
+    { symbol: "B", color: "yellow" },
+    { symbol: "B", color: "blue" },
+    { symbol: "A", color: "cyan" },
+    undefined,
+  ],
+  [
+    { symbol: "D", color: "red" },
+    { symbol: "F", color: "cyan" },
+    { symbol: "B", color: "red" },
+    {
+      symbol: "C",
+      color: "magenta",
+      playedBy: BLACK_PLAYER,
+      lastPlayed: false,
+    },
+    { symbol: "E", color: "yellow" },
+    { symbol: "O", color: "grey", playedBy: null, lastPlayed: false },
+    { symbol: "A", color: "yellow" },
+  ],
+  [
+    undefined,
+    { symbol: "F", color: "yellow" },
+    { symbol: "E", color: "red" },
+    { symbol: "E", color: "blue", playedBy: BLACK_PLAYER, lastPlayed: false },
+    { symbol: "F", color: "green" },
+    { symbol: "D", color: "yellow" },
+    { symbol: "F", color: "magenta" },
+    undefined,
+  ],
+  [
+    undefined,
+    undefined,
+    { symbol: "A", color: "blue" },
+    { symbol: "D", color: "magenta" },
+    {
+      symbol: "A",
+      color: "magenta",
+      playedBy: BLACK_PLAYER,
+      lastPlayed: false,
+    },
+    { symbol: "B", color: "magenta" },
+    { symbol: "B", color: "green" },
+    undefined,
+    undefined,
+  ],
+  [
+    undefined,
+    undefined,
+    undefined,
+    { symbol: "C", color: "blue" },
+    { symbol: "E", color: "cyan", playedBy: BLACK_PLAYER, lastPlayed: false },
+    { symbol: "F", color: "red" },
+    { symbol: "C", color: "red" },
+    undefined,
+    undefined,
+    undefined,
+  ],
+];
 
 describe("renderTile", () => {
   it("should render a tile", () => {
     const tile: Tile = { color: "blue", symbol: "A" };
     expect(renderTile(tile)).toMatch(/A/);
   });
 });
+
+describe("Find siblings around a tile", () => {
+  it("coords should not exist", () => {
+    const exist = checkIfCoordsExist(board, { x: 1, y: 1 });
+    expect(exist).toBeFalsy();
+  });
+  it("coords should exist", () => {
+    const exist = checkIfCoordsExist(board, { x: 1, y: 4 });
+    expect(exist).toBeTruthy();
+  });
```

Comment:
These tests are not testing the `findSiblings` method so they should be moved inside another `describe`

Reply:
Done